README.md

```diff
@@ -1,2 +1,115 @@
-# tau
-TAU (Tizen Advanced UI) project for supporting web app development based on web-based ui widget components library.  This will be replacement and clone of previously existing https://review.tizen.org/gerrit/#/admin/projects/platform/framework/web/tau
+# TAU
+
+The Tizen Advanced UI Framework components allow you to create and manage various kinds of UI components. The components represent a visual UI element, such as a button or slider, which gives you interaction and manipulation features.
+
+## Getting Started
+
+- Clone a copy of the master branch in the TAU Git repository:
+
+`git clone git@github.com:Samsung/TAU.git`
+
+- Change to the tau directory:
+
+`cd TAU`
+
+- Check out the latest version of TAU:
+
+`git checkout tizen_X.Y`
+
+eg. `git checkout tizen_5.0`
+
+### Prerequisites
+
+To build a TAU library, ensure that you have both Git and Node.js (6.16.0) installed.
+
+Example install using `nvm`:
+
+```
+nvm install 6.16.0
+nvm alias default 6.16.0
+```
+
+- Test that you have grunt installed:
+
+`grunt -V`
+
+If grunt is not installed, then run:
+
+`(sudo) npm install -g grunt-cli`
+
+### Installing
+
+- Install the build module by npm:
+
+`npm install`
+
+Build TAU by running the following command in the tau directory:
+
+`grunt build`
+
+## Running the tests
+
+TAU has automated tests. For run tests execute command:
+
+`grunt test`
+
+## Demo application:
+
+UIComponents sample web application demonstrating how to implement UI Components for wearable based on TAU.
+
+Path in repository:
+
+mobile: `examples\mobile\UIComponents`
+
+wearable: `examples\wearable\UIComponents`
+
+- Install http-server
+
+`npm install http-server -g`
+
+- Launch server in app directory:
+
+`http-server -p <port> -a localhost`
+
+eg. `http-server -p 8888 -a localhost`
+
+
+### And coding style tests
```

Comment:
"And" is not needed here, just write
"Coding style tests"

Reply:
Done